### PR TITLE
Remove SQL suffix from form processor models

### PR DESCRIPTION
**Pull request by millerdev**
Review by commit? Should I split this out into a separate PR for each commit?

The underlying SQL tables are not being renamed, although they could be if we thought it was worthwhile.

This PR may conflict with other open PRs, and new occurrences of the old model names will need to be updated (hopefully that would cause tests to fail so it should be obvious).

### Safety story
Renaming all occurrences of model classes in the codebase.

### Automated test coverage

There are many tests that touch these classes.

### QA Plan

No QA.

### Migrations

Some migration files were renamed, but no new migrations were added.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/form_processor/models.py [hunk 1]
```diff
@@ -51,7 +51,7 @@
 XFormOperationSQL_DB_TABLE = 'form_processor_xformoperationsql'
 
 CommCareCaseSQL_DB_TABLE = 'form_processor_commcarecasesql'
```
Comment (snopoke) [inline corehq/form_processor/models.py:53]: why not also rename these other constants?

**Comment by millerdev** [comment]
I think I did the migrations the wrong way. Reworking, which will probably involve a force-push...

File: corehq/form_processor/models.py (hunk 1 above)
Comment (millerdev) [reply corehq/form_processor/models.py]: Thanks, did this during the migration rework.

**Comment by millerdev** [comment]
Do we pickle any of these classes anywhere? If yes, I should add aliases so they can be imported for unpickling.

**Comment by snopoke** [comment]
> Do we pickle any of these classes anywhere? If yes, I should add aliases so they can be imported for unpickling.

Not that I'm aware of. But we do serialize them in the `dump_load` app. It might be an issue for any data that was dumped prior to this change.

**Review by snopoke — APPROVED** [review]

**Pull request merged**